package.json

```diff
@@ -6,7 +6,7 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "prepublishOnly": "npm install && npm run clean && npm run build",
-    "build": "tsc -b",
+    "build": "npm run build:highlight && tsc -b",
```

Comment:
`build:highlight` doesn't need to build every time.

Reply:
oh okay, ill delete it.

Reply:
deleted https://github.com/dimaslanjaka/hexo-util/commit/8663c657d2a231845ddc05f6f59264ccf92e2fa6